core/v1/register.go

```diff
@@ -14,7 +14,6 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-//TODO: this file is going to be moved to k8s.io/api
```

Comment:
@davidstack Thanks for submitting the PR!

However, the code changes to `k8s.io/api` do not go in this repo. This repo is not authoritative and the code in this repo is copied from https://github.com/kubernetes/kubernetes/tree/master/staging/src/k8s.io/api.

Please submit it as a PR to the main `kubernetes/kubernetes` repo and make the changes here: https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/api/core/v1/register.go.

Reply:
ok,i close this commit